autopush/main_argparse.py

```diff
@@ -105,6 +105,10 @@ def add_shared_args(parser):
                         help="AWS DynamoDB endpoint override",
                         type=str, default=None,
                         env_var="AWS_LOCAL_DYNAMODB")
+    parser.add_argument('--allow_table_rotation',
+                        help="Allow monthly message table rotation",
+                        action="store_true", default=False,
```

Comment:
default to False here?

Also this isn't being passed along to AutopushConfig. Annoyingly enough, this still needs a line in AutopushConfig::from_argparse

Reply:
crap, yeah, forgot that.